src/tribler/core/conftest.py

```diff
@@ -64,6 +65,27 @@ def pytest_runtest_protocol(item: Function, log=True, nextitem=None):
         yield
 
 
+@pytest.fixture(autouse=True)
+def ensure_gc():
+    """ Ensure that the garbage collector runs after each test.
+    This is critical for test stability as we use Libtorrent and need to ensure all its destructors are called. """
+    # For this fixture, it is necessary for it to be called as late as possible within the current test's scope.
+    # Therefore it should be placed at the highest possible scope. If there are two or more autouse fixtures within
```

Comment:
> Therefore it should be placed at the highest possible scope

The wording looks a bit misleading to me: according to the Pytest documentation, the highest possible scope is "session", and we want to use the lowest possible scope, "function"

https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

But in "function" scope, we want to be on the first place. So, maybe we should write something like

> Therefore it should be placed at the first place in the "function" scope

Reply:
Done